pages/bare_metal_cloud/virtual_private_servers/starting_with_a_vps/guide.fr-fr.md

```diff
@@ -145,29 +161,39 @@ Vous pouvez également sélectionner une **clé SSH** à installer sur le systè
 > Pour avoir un système d’exploitation **Windows** fonctionnant sur un VPS, il faut l'avoir préalablement choisi **dans le processus de commande**. Un VPS avec un autre OS installé ne peut pas être réinstallé avec Windows via la méthode décrite ci-dessus.
 >
 
-Une barre de progression de l'installation s'affiche dans votre espace client. Veuillez noter que ce processus peut prendre jusqu'à 30 minutes.
+Le processus de réinstallation peut prendre quelques minutes.
```

Comment:
Shouldn't we keep mentioning that it can last up to 30mns ? "Quelques minutes" is usually considered as less than 10mns.

Reply:
Yes Stephane tested in front of me and the process is very quick so he asked me to update it